config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  # get 'root/index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  root "message#index"
+end
```

Comment:
- [ ] Kindly provide a route that will allow you to get the random greeting. in your frontend repo, you are trying to fetch the random greeting  at this endpoint `/api/v1/messages` kindly create this route here 

```rb
  get '/api/v1/messages', to: 'messages#index'

```